src/components/ConsoleComponent.js

```diff
@@ -24,71 +21,102 @@ const useStyles = makeStyles((theme) => ({
     boxSizing: 'border-box'
   },
   button: {
+    marginRight: '20px',
+    color: theme.palette.common.white,
+    textTransform: 'none'
+  },
+  problemsButton: {
     padding: 0,
     color: theme.palette.common.white,
     textTransform: 'none'
   },
   expandIcon: {
     width: '29px' // Lines up with padding
   },
-  warning: {
-    paddingRight: '5px',
-    paddingLeft: '20px',
-    color: 'khaki'
-  },
-  error: {
-    paddingRight: '5px',
-    paddingLeft: '20px',
-    color: 'red'
-  },
-  success: {
-    paddingRight: '5px',
-    paddingLeft: '20px',
-    color: 'green'
-  },
   pre: {
     margin: '0px'
+  },
+  circle: {
+    'border-radius': '50%',
+    'min-width': '22px',
+    height: '22px',
+    paddingLeft: 1,
+    paddingRight: 1,
+    background: '#21121d',
```

Comment:
I think this color looks a bit out of place. Consider trying `#263238`. This is the same color as the background of the code mirror component, and I think it looks good.

Reply:
Additionally, I think it might work well if there is still some sort of color based indication that something went wrong. Maybe if there are errors, the problem count could be red, and if only warnings, yellow. Similar to how we do it in SUSHI. If you try this and think it looks bad though, feel free to shut this down.

Reply:
For consistency, maybe we could use something like this: 
![image](https://user-images.githubusercontent.com/52747882/134979563-d4f60f91-85de-4889-89be-36de31e59a54.png)

since we already do that on invalid JSON.

Reply:
This is a really good idea! I've toyed with the colors a bit in `4388dc0`, adding highlighting based on the error cases that you described (yellow for only warnings, red for errors, white for no errors). Let me know if you think it looks off at all, or perhaps if you think it makes sense to introduce a green color in some way to indicate a successful run.